src/cdk/testing/component-harness.ts

```diff
@@ -25,6 +25,9 @@ export type AsyncOptionPredicate<T, O> = (item: T, option: O) => Promise<boolean
 export type HarnessQuery<T extends ComponentHarness> =
     ComponentHarnessConstructor<T> | HarnessPredicate<T>;
 
+/** Queries that can be passed into a locator function. */
+export type LocatorQueries = [HarnessQuery<any> | string, ...(HarnessQuery<any> | string)[]];
```

Comment:
I'm not a fan of the name, but I couldn't come up with something better. I also don't want to do it inline since it's a little long.